pixiedust/display/chart/renderers/bokeh/histogramDisplay.py

```diff
@@ -54,20 +54,26 @@ def getChartOptions(self):
         ]
 
     def canRenderChart(self):
-        valueFields = self.getValueFields()
-        if len(valueFields) != 1:
-            return (False, "You can only select one value")
-        else:
-            return (True, None)
+        return (True, None)
 
     def getExtraFields(self):
         color = self.options.get("color")
         return [color] if color is not None else []
     
-    def createBokehChart(self):     
-        
+    def createBokehChart(self):
+        plotwidth = int(self.getPreferredOutputWidth()/2)
+        plotheight = int(self.getPreferredOutputHeight() * 0.75)
         binsize = int(self.options.get('binsize', 10))  
-        return Histogram(self.getWorkingPandasDataFrame(), values=self.getValueFields()[0],
-            plot_width=800,bins=binsize, color=self.options.get("color"), 
-            xgrid=True, ygrid=True
-        )
\ No newline at end of file
+        valueFields = self.getValueFields()
+        histograms = []
+        if len(valueFields) != 1:
+            for i in valueFields:
+                histograms.append(Histogram(self.getWorkingPandasDataFrame(), values=i,
+            plot_width=plotwidth, plot_height=plotheight,bins=binsize, color=self.options.get("color"), 
+            xgrid=True, ygrid=True))
+
+            return histograms
+        else:
+            return Histogram(self.getWorkingPandasDataFrame(), values=self.getValueFields()[0],
+            bins=binsize, color=self.options.get("color"), 
```

Comment:
@DTAIEB for single value histogram, bokehBaseDisplay is overridding width or height:

if not isinstance(charts, list):
            charts.add_tools(ResizeTool())
            charts.title = self.options.get("title", "")
            charts.plot_width = int(self.getPreferredOutputWidth() - 10 ) <--
            charts.plot_height = int(self.getPreferredOutputHeight() - 10  )  <--

I found it confusing to keep the plot_width and plot_height in the input. However I intent to see 4xx x 4xx chart - it looks more neat, less pixelized. What is your suggestion? I don't mean to over complicate just use values set in bokehBaseDisplay.